.github/workflows/build-swift.yml

```diff
@@ -0,0 +1,143 @@
+---
+name: Build Swift Package
+
+on:
+  push:
+    branches:
+      - "rc"
+      - "hotfix-rc-swift"
+    paths:
+      - "languages/swift/**"
```

Comment:
We expect to rarely edit these files.

Reply:
What is a better path to trigger this workflow on?

Reply:
I think triggering it on everything might make more sense. Changes to the rust code will change the output of the workflow.

Reply:
Done!